_posts/2024-09-25-2023-2024-report.md

```diff
@@ -0,0 +1,143 @@
+---
+
+Founded in September of 2020, Brink has officially made it through our first
+Bitcoin 4-year cycle. As we celebrate our 4 year anniversary, we have compiled
+our first annual report highlighting engineering accomplishments in 2023,
+engineer's progress and plans during 2024, and details of Brink's fundraising
+and expenses.
+
+We hope this report provides tangible insights into progress made on Bitcoin's open
+source codebase, by Brink engineers, that is often hard to assess or quantify.
+
+We provide some highlights from the report below, with the [full report
+available here][2023 annual report].
+
+<a href="/assets/files/brink-annual-report-2023.pdf" target="_blank"><img
+src="/assets/images/brink-report-2023-2024.png" alt="Brink's 2023-2024 Annual
+Report Preview Page" style="max-height:100%" /></a>
+
+## Organizational Updates
+
+### Fundraising
+
+Brink’s sponsors donated approximately $2.4 million toward our mission last year.
+These contributions came from over 500 different donors.
+
+> _Brink is thankful for our diverse donor base, which includes high-net-worth
+individuals, Bitcoin miners, Bitcoin mining pool operators, venture capitalists,
+Lightning businesses, hardware device manufacturers, family offices, exchanges,
+and hundreds of individual donors._
+
+In the [report][2023 annual report], we list all of our major donors, who
+contributed over $5,000 each, and thank them for the crucial role they played in
+our success.
+
+### Expenses
+
+In 2023, Brink's expenses were approximately $1,600,000. The summary of these
+expenses include:
+
+* Program: Developer Funding ($1,220,000)
+  * Developer Salaries & Grants ($1,100,000)
+  * Travel ($35,000)
+  * Office ($85,000)
+* Program: General Bitcoin Core Support ($16,000)
+* Program: Bitcoin Core Developer Meetings ($41,000)
+* Program: Bitcoin Optech ($14,000)
+* Operations, Staff, and Fundraising ($309,000)
+  * Staff Compensation ($200,000)
+  * Operational expenses ($92,000)
+  * Fundraising ($17,000)
+
+We breakdown these items in more detail in the [report][2023 annual report].
+
+_Note: For clarity, the figures provided are approximations based on the cash
+method for 2023. Audited, accrual-based GAAP financials will be available in our
+public 2023 Form 990 filing._
+
+## Engineering Highlights
+
+We focus on the quantity and quality of each engineer's code review and
+testing before we look at any other contributions they made.  This is a
+deliberate decision.  We frequently hear that many open source
+contributors worry that they will only receive offers if they pursue
+headline-grabbing projects, rather than quietly performing the
+high-quality code reviews and testing that are essential to keeping
+Bitcoin Core secure and getting useful changes merged quickly.  As an
+organization, Brink always puts review and testing first and foremost,
+and we've tried to reflect that part of our internal culture in this
+external [report][2023 annual report].
```

Comment:
Maybe add somewhere "While number of review comments is far from the perfect metric, we noticed how uncommonly large these numbers were, and wished to highlight them as a reflection of Brink engineers' focus on review and security" or something?

For reference, 7 people's 5100 review comments is 17.5% of the 29014 total (according to the year survey).

Reply:
Added, thanks!